src/core/SpyStrategy.js

```diff
@@ -150,10 +150,13 @@ getJasmineRequireObj().SpyStrategy = function(j$) {
    * @name SpyStrategy#throwError
    * @since 2.0.0
    * @function
-   * @param {Error|String} something Thing to throw
+   * @param {Error|Object|String} something Thing to throw
    */
   SpyStrategy.prototype.throwError = function(something) {
-    var error = something instanceof Error ? something : new Error(something);
+    var error =
+      something instanceof String || typeof something === 'string'
```

Comment:
Please use j$.isString_ to do the type check.

Reply:
Done, thanks for the suggestion